src/psyclone/tests/test_files/dynamo0p3/invalid_builtins_mod.f90

```diff
@@ -140,20 +140,6 @@ module dynamo0p3_builtins_mod
      procedure, nopass :: inc_X_divideby_Y_code
   end type inc_X_divideby_Y
 
```

Comment:
Worth adding one that has an invalid mixture of data types?

Reply:
I'd rather leave it for #1107 so I can add a proper check that mixing is only allowed for the conversion built-ins. I added a "TODO" in the file.